tests/Controllers/RegisterUserControllerTest.php

```diff
@@ -0,0 +1,33 @@
+<?php
+/**
+ * Created by PhpStorm.
+ * User: academy
+ * Date: 26/11/2018
+ * Time: 15:40
+ */
+
+namespace Tests\Controllers;
+
+use PHPUnit\Framework\TestCase;
+use Portal\Controllers\RegisterUserController;
+use Portal\Models\UserModel;
+use Slim\Http\Request;
```

Comment:
I don't think that the request or response are being used here

Reply:
fixed in b9c7cf8